src/lib/Utils.java

```diff
@@ -0,0 +1,7 @@
+package lib;
+
+public class Utils {
+	public static double deadband(double value, double deadband, double center){

```

Comment:
This needs a comment explaining what this does and what it is for.  You should also have tests.